Mathlib/RingTheory/MvPowerSeries/Trunc.lean

```diff
@@ -0,0 +1,107 @@
+/-
+Copyright (c) 2019 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Kenny Lau
+-/
+
+import Mathlib.RingTheory.MvPowerSeries.Basic
+import Mathlib.Data.Finsupp.Interval
+
+/-!
+
+# Formal (multivariate) power series - Truncation
+
+`trunc n φ` truncates a formal power series to the polynomial
+that has the same coefficients as `φ`, for all `m < n`, and `0` otherwise.
```

Comment:
```suggestion
`trunc n φ` truncates a multivariate formal power series `φ` to the polynomial
that has the same coefficients as `φ`, for all `m < n`, and `0` otherwise.
```
Perhaps a word can be said about the ordering, since these `m` and `n` are not in `ℕ`?